ols/utils/config.py

```diff
@@ -35,7 +35,9 @@ def load_config_from_env() -> None:
     logger_config = config_model.LoggerConfig()
     ols_config.logger_config = logger_config
 
-    logger_config.default_level = os.getenv("LOG_LEVEL", logging.INFO)
+    logger_config.default_level = os.getenv(
+        "LOG_LEVEL", logging.getLevelName(logging.INFO)
```

Comment:
What about just `os.getenv("LOG_LEVEL", "info")`? Would that work too?

The function actually returns different things based on the input. I guess those are all valid values, but its funny.
![image](https://github.com/openshift/lightspeed-service/assets/37311499/0eb22312-c3cc-4a68-9c4f-7be348882697)

Reply:
just bare "info" will introduce yet another anonymous magic constant :/

Reply:
Ok.

Reply:
it's surprising that logging doesn't define its own string constants for "INFO", "DEBUG", etc..but i guess they only chose to define numeric constants.

Anyway @tisnik i think you've made the right call here to avoid us defining another constant.  This ties the behavior/constants to the core logging library expectations